ArcGISRuntimeSDKQt_CppSamples/Analysis/Geotriggers/Geotriggers.cpp

```diff
@@ -28,12 +29,20 @@
 #include "GeotriggersTypes.h"
 #include "FeatureFenceParameters.h"
 #include "LocationGeotriggerFeed.h"
+#include "LocationDisplay.h"
 #include "Map.h"
 #include "MapQuickView.h"
+#include "MapViewTypes.h"
 #include "PortalItem.h"
 #include "ServiceFeatureTable.h"
 #include "SimulatedLocationDataSource.h"
 #include "SimulationParameters.h"
+#include "TaskWatcher.h"
+#include "ArcGISFeature.h"
+#include "AttributeListModel.h"
+#include "Attachment.h"
+#include "AttachmentListModel.h"
```

Comment:
These are not alphabetical and they should be. @GuillaumeBelz I though our team's git hooks ran against the samples as well. Is that not the case anymore?